### ISSUE TITLE
[Feature] 

### ISSUE THREAD

**Issue by jeannotes** [open-mmlab/mmclassification#1337]
### Branch

master branch (0.24 or other 0.x version)

### Describe the feature

in mmcls's visualizationHook, 
 https://github.com/open-mmlab/mmclassification/blob/c7ec630c37dd2c7c997405a70b9fafa6a54e5f95/mmcls/engine/hooks/visualization_hook.py#L9
and 
https://github.com/open-mmlab/mmclassification/blob/c7ec630c37dd2c7c997405a70b9fafa6a54e5f95/mmcls/engine/hooks/visualization_hook.py#L42
should this be changed to 
`
from mmcls.visualization import ClsVisualizer

self._visualizer: ClsVisualizer = ClsVisualizer.get_current_instance()
`

### Will you implement it?

- [ ] I would like to implement this feature and create a PR!

**Comment by mzr1996** [comment]
No, please refers to https://mmengine.readthedocs.io/zh_CN/latest/api/generated/mmengine.visualization.Visualizer.html#mmengine.visualization.Visualizer.get_instance
![图片](https://user-images.githubusercontent.com/26739999/215966076-00802beb-b1af-4a9d-afec-c7a5d86ac6c5.png)
